res/blackbox-tests/configuration.toml

```diff
@@ -95,4 +95,4 @@ Type="http"
 
 [Logging]
 EnableRemote = false
-File = "./logs/app-service-configurable-blackbox-tests.log"
+File = ""
```

Comment:
Should we not just remove this entry all together? why keep it blank?

Reply:
So users know it exists. This is consistent with core services.